style.css

```diff
@@ -0,0 +1,199 @@
+body {
+    box-sizing: border-box;
+    padding: 0;
+    margin: 0;
+    font-family: 'Lato', sans-serif;  
+    background-color: #EEE;
+}
+#header {
+    position : fixed;
+    top: 0;
+    min-height: 75px;
+    padding: 0px 20px;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    background-color: #eee;
+    width: 100%;
+}
+
+#logo{
+    width: 30vw;
+
+}
+
+#logo img {
+    width: 100%;
+    height: 100%;
+    min-width: 300px;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    text-align: center;
+    margin: 20px;
+}
+
+#nav-bar{
+    font-weight: 400;
+}
+
+nav ul {
+    width: 30vw;
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+}
+
+
+li {
+    list-style: none;
+}
+
+a {
+    text-decoration: none;
+    color: black;
+}
+
+
+.Center{
+    position : relative;
+} 
+
+
+.form{
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    text-align: center;
+    height: 200px;
+    margin-top: 100px;
+}
+
+.form h1{
+    margin-bottom: 20px;
+    word-wrap: break-word;
+}
+
+
+.Content-center{
+    max-width: 1000px;
+    width: 100%;
+    margin: 0 auto;
+}
+
+#features{
+    margin-top: 20px;
+}
+
+
+#icon{
+    color: orange;
+    font-size: 50px;
+}
+
+button{
+    background-color: orange;
+    border: none;
+    padding: 4px;
+    margin-top: 10px;
+    width: 130px;
+    height: 30px;
+    font-weight: 1000;
+    margin-bottom: 50px;
+    cursor: pointer;
+}
+
+#email{
+    padding: 5px;
+    width: 220px;
+}
+
+.description{
+    display: flex;
+    flex-direction: column;
+    margin-left: 50px;
+    justify-content: center;
+    height: 125px;
+    width: 80vw;
+    padding: 5px;
+}
+
+.description h2, .description p {
+    margin: 0;
+}
+
+.icon{
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-right: 30px;
+    height: 125px;;
+}
+
+iframe{
+    width: 600px;
+    height: 400px;
+    margin: 0 auto;
+}
+
```

Comment:
This part of project should be centered in its parent div, please, try to change it now